File: example_code/python/TLS.py
import MySQLdb
import sys
import platform

kwargs = {
}
kwargs["host"]=sys.argv[1]
kwargs["port"]=4000
kwargs["user"]=sys.argv[2]
kwargs["password"]=sys.argv[3]
kwargs["database"]="test"
kwargs["ssl"]={"ca": sys.argv[4]}
if platform.system() != 'Windows':
  kwargs['ssl_mode'] = "VERIFY_IDENTITY",
connection = MySQLdb.connect(**kwargs)

with connection:
  with connection.cursor() as cursor:
    cursor.execute("SELECT DATABASE();")
    m = cursor.fetchone()
    print(m[0])

File: example_code/node/TLS.js
var mysql = require('mysql2');



var connection = mysql.createConnection({
  host:  process.argv[2],
  port: 4000,
  user: process.argv[3],
  password: '1234567',
  database: 'test',
  ssl: {
    minVersion: 'TLSv1.2',
    rejectUnauthorized: true
  }
});
connection.connect(function(err) {
  if (err) {
    throw err
  }
  connection.query('SELECT DATABASE();', function(err, rows) {
    if (err) {
      throw err
    }
    console.log(rows[0]['DATABASE()']);
    connection.end()
  });
});